### PR TITLE
Issue #SB-30214 fix: content upload and review APIs issue fixes

### PR DESCRIPTION
**Pull request by vaibhavbhuva**

### CODE REVIEW

File: server/proxyUtils.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const envVariables = require('./config/environment');
 
 const decoratePublicRequestHeaders = function () {
   return function (proxyReqOpts, srcReq) {
-    proxyReqOpts.headers["Content-Type"] = "application/json";
+    // proxyReqOpts.headers["Content-Type"] = "application/json";
```
Comment (vinukumar-vs) [inline server/proxyUtils.js:5]: Why did you remove this? Always our response will be in JSON format. This can't be changed at all.

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

File: server/proxyUtils.js (hunk 1 above)
Comment (vaibhavbhuva) [reply server/proxyUtils.js:5]: This header overrides `reqAsBuffer` and `reqBodyEncoding` due to this content uplaod API is failing.

In the backend application, `Content-type` is not required anymore as we are passing required content-type from the fronted application and appending required headers in the backend.

For more information check link below:
https://www.npmjs.com/package/express-http-proxy#:~:text=))%3B-,parseReqBody,-The%20parseReqBody%20option
